lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -111,11 +125,18 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_TIME, M
    * UpdateNotifierService.notify();
    */
   this.notify = () => {
-    return ModalService.open({
-      template: './components/update-notifier/templates/update-notifier-modal.tpl.html',
-      controller: 'UpdateNotifierController as modal',
-      size: 'update-notifier'
-    }).result;
+    return this.getLatestVersion().then((version) => {
+      return ModalService.open({
+        template: './components/update-notifier/templates/update-notifier-modal.tpl.html',
+        controller: 'UpdateNotifierController as modal',
+        size: 'update-notifier',
+        resolve: {
+          options: _.constant({
+            version: version
+          })
+        }
+      }).result;
+    }).catch(ErrorService.reportException);
```

Comment:
Lets move this `.catch()` to the code that actually calls `.notify()` (which I believe is in `lib/gui/app.js` only)